e-learning-webapps/src/main/webapp/vue-app/main.js

```diff
@@ -11,7 +12,7 @@ const lang = eXo && eXo.env && eXo.env.portal && eXo.env.portal.language || 'en'
 
 const url = `${eXo.env.portal.context}/${eXo.env.portal.rest}/i18n/bundle/locale.addon.elearning-${lang}.json`;
 
-
+export const EventBus = new Vue();
```

Comment:
let's call it tutorialsApp
```suggestion
export const tutorialsApp = new Vue();
```